GameAI/Entity/GoalBehaviour/Composite/Think.cs

```diff
@@ -31,12 +31,14 @@ private Goal<Vehicle> FindNewGoal()
 
             Team otherTeam = this.world.Teams.Values.First(team => team.Colour != this.Owner.Team.Colour);
 
-            switch (this.random.Next(0, 3))
+            switch (this.random.Next(0, 10) % 4)
```

Comment:
Waarom is dit veranderd? 10 is geen meervoud van 4 dus ga je vaker 2 en 3 vaker krijgen als case dan de anderen